Tools/Scripts/libraries/webkitscmpy/webkitscmpy/contributor.py

```diff
@@ -111,6 +111,9 @@ def add(self, contributor):
                 raise ValueError("'{}' is not a Contributor object".format(type(contributor)))
 
             result = self.create(contributor.name, *contributor.emails)
+            if not result:
+                return None
+
```

Comment:
Why is it OK to silently discard a malformed contributor?

Reply:
Other options here are:
- Log an error but continue
- Raise an exception

In practice, I think both of them would be ok, although would prefer "log an error" since I fear we could otherwise find ourselves in a death spiral if someone managed to commit a bad contributor to contributors.json